obspy/io/css/core.py

```diff
@@ -7,11 +7,14 @@
 from future.builtins import *  # NOQA
 
 import os
+import sys
 
 import numpy as np
 
 from obspy import Stream, Trace, UTCDateTime
 from obspy.core.compatibility import from_buffer
+from obspy.core.util.deprecation_helpers import \
+    DynamicAttributeImportRerouteModule
```

Comment:
This comes from an old copy of this file and was removed in master. Please remove.